cli/src/commands/git.rs

```diff
@@ -392,6 +398,14 @@ fn get_default_fetch_remotes(
     }
 }
 
+fn get_all_remotes(git_repo: &git2::Repository) -> Result<Vec<String>, CommandError> {
+    let git_remotes = git_repo.remotes()?;
+    Ok(git_remotes
+        .iter()
+        .filter_map(|x| x.map(|x| x.to_owned()))
+        .collect())
```

Comment:
If itertools are available, you could use `into_vec()`

Reply:
Didn't get it. Reworked to use `ToOwned` directly

Reply:
I meant [this](https://docs.rs/itertools/0.11.0/itertools/trait.Itertools.html#method.collect_vec), but since this wasn't blocking anyway use either version.